git-extra/getgit

```diff
@@ -13,6 +13,7 @@ function usage(){
     echo "  -k                   keep all downloaded, extracted files in \"/tmp\""
     echo "  -s                   skip downloading"
     echo "  -f                   forcibly update no matter what version it is"
+    echo "  -e                   don't modify the environment (no /cmd in path, /etc/profile.d)"
```

Comment:
"no /cmd in path" sounds like that part is only about not modifying `$PATH`, whereas the code doesn't create `/cmd` at all.